recipes-wlroots/wayvnc/neatvnc_0.6.0.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Neatvnc vnc c library"
```

Comment:
[meta-toradex](https://github.com/toradex/meta-toradex-tezi/blob/kirkstone-6.x.y/recipes-graphics/neatvnc/neatvnc_git.bb) also has a recipe for `neatvnc`.